internal/git/git.go

```diff
@@ -1,27 +1,97 @@
-package git
+package main
```

Comment:
Change back to `git` for use as a module.
Did you change this for testing with `func main()`?

Reply:
Yea just for testing :)